src/server/cli/build-settings.ts

```diff
@@ -48,9 +50,10 @@ export default function buildSettings(config: object, options: ServerOptions, au
     ...options
```

Comment:
Here happens override of config values from `ServerOptions` which are passed from CLI.